[2021-03-10 06:48:09,650] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.storing_user 2021-03-10T06:46:19.464151+00:00 [queued]>
[2021-03-10 06:48:09,655] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.storing_user 2021-03-10T06:46:19.464151+00:00 [queued]>
[2021-03-10 06:48:09,655] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 06:48:09,655] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-10 06:48:09,655] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 06:48:09,665] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): storing_user> on 2021-03-10T06:46:19.464151+00:00
[2021-03-10 06:48:09,667] {standard_task_runner.py:51} INFO - Started process 40241 to run task
[2021-03-10 06:48:09,672] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'data_processing', 'storing_user', '2021-03-10T06:46:19.464151+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_processing.py', '--cfg-path', '/tmp/tmpzc9_b2lj']
[2021-03-10 06:48:09,672] {standard_task_runner.py:76} INFO - Job 14: Subtask storing_user
[2021-03-10 06:48:09,795] {logging_mixin.py:103} INFO - Running <TaskInstance: data_processing.storing_user 2021-03-10T06:46:19.464151+00:00 [running]> on host airflowvm
[2021-03-10 06:48:09,922] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_processing
AIRFLOW_CTX_TASK_ID=storing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-03-10T06:46:19.464151+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-10T06:46:19.464151+00:00
[2021-03-10 06:48:09,922] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-10 06:48:09,923] {bash.py:158} INFO - Running command: echo -e ".separator "," 
. import /tmp/users.csv userstwos" | sqlite3 /home/airflow/airflow/airflow.db
[2021-03-10 06:48:09,927] {bash.py:169} INFO - Output:
[2021-03-10 06:48:09,930] {bash.py:173} INFO - /tmp/users.csv:1: expected 3 columns but found 6 - extras ignored
[2021-03-10 06:48:09,933] {bash.py:177} INFO - Command exited with return code 0
[2021-03-10 06:48:09,945] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=data_processing, task_id=storing_user, execution_date=20210310T064619, start_date=20210310T064809, end_date=20210310T064809
[2021-03-10 06:48:09,956] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-10 06:48:09,973] {local_task_job.py:118} INFO - Task exited with return code 0
